guides/introduction/Embedded Schemas.md

```diff
@@ -92,57 +91,110 @@ end
 defmodule UserProfile do
   use Ecto.Schema
 
-  embedded_schema "profile" do
-    field :age, :integer
-    field :favorite_color, Ecto.Enum, values: [:red, :green, :blue, :pink, :black, :orange]
-    field :avatar_url, :string
+  embedded_schema do
+    field :online, :boolean
+    field :dark_mode, :boolean
+    field :visibility, Ecto.Enum, values: [:public, :private, :friends_only]
   end
 end
 ```
 
-Ta-da! Neat. Note we replaced the `embeds_one` macro by `embedded_schema`: `embeds_one` and `embeds_many` function like fields, similar to relations like `has_many`.
-
-### Writing the migration
+It is important to remember that `embedded_schema` has many use cases independent of `embeds_one` and `embeds_many`. There is a clear relationship between them, but keep in mind that `embedded_schema` has broader applications, like in the intermediate-state use case from the introduction.
```

Comment:
Added a NB so readers recall that embedded_schema is not _only_ useful with embedded data. Something that I surmise trips up new Ecto users.

Reply:
Yes, so from what I said above, I would expand on those use cases here, and not in the intro. :)